lightning_server.cc

```diff
@@ -1,5 +1,6 @@
 #include "config_parser.h"
 #include "lightning_server.h"
+#include "server_config.h"
```

Comment:
As Ky-Cuong mentoned in server_config, we should keep to our convention of includes in alphabetical order.